src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CalculatorTest {
+
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @AfterEach
+    void tearDown() {
+    }
+
+    @Test
+    void add() {
+        assertEquals(7, calculator.add(3, 4));
+    }
+
+    @Test
+    void subtract() {
+        assertEquals(3, calculator.subtract(7,4));
+    }
+
+    @Test
+    void multiply() {
+        assertEquals(26,calculator.multiply(13,2));
+    }
+
+    @Test
+    void divide() {
+        assertEquals(2, calculator.divide(6, 3));
```

Comment:
1 unit test -  1 call to testing method.

1 unit test can have multiple assert:
RiderEntity actual = service.someMethod(arg);

assertNotNull(actual);
assertEquals("Rider", actual.getName);
assertTrue(actual.isValid());

etc.

So multiple asserts could just examine different sides of SINGLE output / result.

But 1 unit test cover 1 scenario.

Your test cover 2 scenarios - postive and negative, and therefore have 2 method calls.

So there should be two unit tests:

@Test
void shoudlCorrectlyDivide() {
 assertEquals(5, calculator.divide(25, 5));
}

@Test
void shouldThrowExceptionWhenDivideByZeroIsPerformed() {
   IllegalArgumentException ex = assertThrows(IllegalArgumentException.class, () -> {calculator.divide(3,0)});
   
   assertEquals("message...", ex.getMessage());
 }
 
 Don't forget to test exception message.